.github/workflows/crowdin.yml

```diff
@@ -0,0 +1,31 @@
+name: Crowdin Action
+
+on:
+  push
+  #push:
+  #  branches: [ master ]
+
+jobs:
+  synchronize-with-crowdin:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: crowdin action
+        uses: crowdin/github-action@v1
+        with:
+          upload_sources: true
+          upload_translations: false
+          download_translations: true
+          localization_branch_name: l10n_crowdin_translations

```

Comment:
I think this should be l18n, as the folder name of the translations is also l18n